en/flying/led_meanings.md

```diff
@@ -0,0 +1,38 @@
+# LED meanings
+
+[Pixhawk flight controllers](../flight_controller/pixhawk_series.md) have an RGB LED to indicate the current status of the FMU.
+
+![LED meanings](../../images/led_meanings.gif)
+
+> **Note:** **GPS unit vs GPS Lock** 
+Please note that Pixhawk’s RGB LED will only turn green and allow you to execute guided mission only if you have a GPS unit and Pixhawk is able to validate the global position from the GPS unit. If you have a GPS unit plugged in but the RGB LED continues to be blue, verify that Pixhawk is reading your GPS properly, and that your GPS is sending a proper GPS position.
+    
+
+* **[Solid Blue] Armed, No GPS Lock:** Indicates Pixhawk has been armed and has no position lock from a GPS unit.
+When Pixhawk is armed, it will unlock control of the motors, allowing you to fly your drone.
+As always, exercise caution when arming, as large propellers can be dangerous at high revolutions.
+Pixhawk cannot perform guided missions in this mode.
+
+* **[Pulsing Blue] Disarmed, No GPS Lock:** Similar to above, but your Pixhawk is disarmed.
+This means you will not be able to control motors, but all other subsystems are working.
+
+* **[Solid Green] Armed, GPS Lock:** Indicates Pixhawk has been armed and has a valid position lock from a GPS unit.
```

Comment:
1. Is it Pixhawk or PX4 that we should be referring to?
1. We can infer that the GPS comes from the GPS unit. So "Indicates Pixhawk is armed and has a valid position lock."

Reply:
Should I replace "Pixhawk" with "vehicle"? so it will make more sense if it says "vehicle is armed"

Reply:
Yes, I think "vehicle" is better.

Reply:
I'm not sure. I think vehicle is better here.